.github/workflows/main.yml

```diff
@@ -65,12 +65,26 @@ jobs:
       # Compilation and registration of the PyCOMTest server dll
       - name: Set up MSVC
         uses: microsoft/setup-msbuild@v2
+
+      # See https://github.com/actions/runner-images/issues/9701
+      # Adapted from https://github.com/actions/runner-images/issues/9873#issuecomment-2139288682
+      - name: Install missing Visual Studio components
+        run: |
+          Set-Location "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+          $VsInstallPath = vswhere.exe -latest -products * -requires Microsoft.Component.MSBuild -property installationPath
+          [string]$ComponentsToAdd = @(
+            "Microsoft.VisualStudio.Component.VC.14.29.16.11.ATL"
+          ) | ForEach-Object {"--add $_"}
+          $ArgumentList = ('modify', '--installPath', "`"$VsInstallPath`"", $ComponentsToAdd, '--quiet', '--norestart', '--nocache')
+          echo "vs_installer.exe $($ArgumentList -join ' ')"
+          # should be run twice for some reason
+          Start-Process -FilePath vs_installer.exe -ArgumentList $ArgumentList -Wait -PassThru -WindowStyle Hidden
+          Start-Process -FilePath vs_installer.exe -ArgumentList $ArgumentList -Wait -PassThru -WindowStyle Hidden
```

Comment:
I couldn't find an explanation in any of the linked issues as to why we need to run this twice. But I confirmed it's needed. It sure doesn't help with the extra time it takes.